### PR TITLE
Add Permissions and Authorization Videos

### PR DESCRIPTION
**Pull request by reidsn**
- Adds permissions and authorizations videos to Near Me wiki

### CODE REVIEW

File: content/en/nearme/demo_videos/videos.md [hunk 1]
```diff
@@ -37,3 +37,15 @@ In this video, we look through the different ways to observe your Near Me Virtua
 In this video, we look through the ways your can customize and improve your Near Me Virtual Agent.
 
    <iframe width="560" height="315" src="https://www.youtube.com/embed/sWhigyIh3Gw" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen style="border-radius: 30px;"></iframe>
+
+## Authorize Seasalt.ai on Google Business Profile
+
+In this video, we show the process of authorizing Seasalt.ai to access your virtual agent through Google Business Profile.
+
+   <iframe width="560" height="315" src="https://www.youtube.com/embed/Y8x0ge51R4U" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (amy17519) [inline content/en/nearme/demo_videos/videos.md:45]: Can you please copy the style from other widget? At least the border radius is missing ha

**Review by amy17519 — CHANGES_REQUESTED** [review]

File: content/en/nearme/demo_videos/videos.md (hunk 1 above)
Comment (reidsn) [reply content/en/nearme/demo_videos/videos.md]: Oops, yep, missed that!

**Review by amy17519 — APPROVED** [review]
Thanks!

File: content/en/nearme/demo_videos/videos.md [hunk 2]
```diff
@@ -37,3 +37,15 @@ In this video, we look through the different ways to observe your Near Me Virtua
 In this video, we look through the ways your can customize and improve your Near Me Virtual Agent.
 
    <iframe width="560" height="315" src="https://www.youtube.com/embed/sWhigyIh3Gw" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen style="border-radius: 30px;"></iframe>
+
+## Authorize Seasalt.ai on Google Business Profile
+
+In this video, we show the process of authorizing Seasalt.ai to access your virtual agent through Google Business Profile.
```
Comment (xuchen) [inline content/en/nearme/demo_videos/videos.md:43]: can you say more about *why* businesses want to do that?

File: content/en/nearme/demo_videos/videos.md [hunk 3]
```diff
@@ -37,3 +37,15 @@ In this video, we look through the different ways to observe your Near Me Virtua
...
+
+   <iframe width="560" height="315" src="https://www.youtube.com/embed/Y8x0ge51R4U" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen style="border-radius: 30px;></iframe>
+
+## Google Permissions in Near Me Messaging
+
+In this video, one of our Natural Language Processing Engineers will talk through the Google permissions required by Near Me Messaging.
```
Comment (xuchen) [inline content/en/nearme/demo_videos/videos.md:49]: Better give a screenshot of the extra permissions asked, then explain them too.

**Review by xuchen — CHANGES_REQUESTED** [review]

**Review by xuchen — APPROVED** [review]
thanks!

**Pull request merged**